imsim/lsst_image.py

```diff
@@ -266,9 +268,24 @@ def addNoise(self, image, config, base, image_num, obj_num, current_var, logger)
             logger.info("Applying vignetting according to radial spline model.")
             radii = Vignetting.get_pixel_radii(camera[det_name])
             sky.array[:] *= self.vignetting.apply_to_radii(radii)
-
+            
+        if self.apply_fringing:
+            # get det number
+            det_num = base['output']['det_num']['current'][0]
+            # get boresight ra/dec
+            boresight = [self.boresight.ra.deg,self.boresight.dec.deg]
```

Comment:
Don't do this.  Let CCD_Fringing take a CelestialCoord object for its boresight parameter, rather than using an adhoc data structure (i.e. the list of ra, dec, both in degrees).